app/controllers/project_comments_controller.rb

```diff
@@ -144,7 +146,9 @@ def destroy
               project: @project.name
             )
           )
-          render json: {}, status: :ok
+          render json: { linked_id: @project.id,
+                         counter: @project.project_comments.count },
```

Comment:
Same here, add comment `# Used for counter badge` or something along those lines.